pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -130,3 +137,21 @@ func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder
 	configMap.Data["version"] = version.Get().String()
 	return resourceapply.ApplyConfigMap(client, recorder, configMap)
 }
+
+func manageServiceAccountCABundle(lister corev1listers.ConfigMapLister, client coreclientv1.ConfigMapsGetter, recorder events.Recorder) (*corev1.ConfigMap, bool, error) {
```

Comment:
This can now be reduced to a resourceSync against `operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "kube-apiserver-server-ca"`